api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -2382,8 +2381,6 @@ public void testShareWorkspaceAddBillingProjectUser() {
     ShareWorkspaceRequest shareWorkspaceRequest =
         new ShareWorkspaceRequest()
             .workspaceEtag(workspace.getEtag())
-            .addItemsItem(
```

Comment:
we no longer need to specify current users in share requests to ensure they are not removed.  There are a few other changes like this here.